client/src/app.tsx

```diff
@@ -74,7 +74,16 @@ function DocumentOrPageNotFound(props) {
   );
 }
 
+function Content({ pathname, appProps }) {
+  if (/\.(png|webp|gif|jpeg|svg)$/.test(pathname)) {
```

Comment:
Think the `e` is maked optional in some of the other regex to support `jpeg` and `jpg`
```suggestion
  if (/\.(png|webp|gif|jpe?g|svg)$/.test(pathname)) {
```

Reply:
LOL, just saw https://github.com/mdn/yari/pull/2300 after

Reply:
> LOL, just saw https://github.com/mdn/yari/pull/2300 after

Yup :D Thanks for catching it here too. I'd meant to amend the commit, but forgot to get back to it...